apache/dubbo-kubernetes
# Feature/Console/Service Test

## What is the purpose of the change

related to [ISSUES#84](https://github.com/apache/dubbo-kubernetes/issues/84)
In order to support the service testing function of the console, it is necessary to support the user's protobuf definition and generate it based on grpc-gen-protoc
Descrptor returns the data structure of the original .proto file.

For example: The user requests ```http:127.0.0.1:8081/com.xxx.xxx.XXService``` and returns the corresponding ```.proto``` file.

The intention of this test case: start a grpcServer, register the dynamic proto file, the client initiates a parsing request to the server, and compares it with the expected service list to determine whether it is consistent.

## How to verify this change

- testFileByFilename: used to verify whether the proto content returned by grpcserver after loading the service meets expectations.
- testListServices: used to verify whether grpcServer successfully loaded the service list
